crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -811,6 +811,7 @@ en:
     not_found: 'Not found'
     chef_server_down: 'Chef server is not available'
     param_missing: 'A required parameter is missing'
+    before_install: 'Some of the API is not available before installation'
```

Comment:
Well, my proposal was ambiguous (kind of on purpose ;-)) because it wasn't saying if this was "This API" or "API in general". If you want to be more specific, I'd just say "This API".